piccolo_admin/endpoints.py

```diff
@@ -865,11 +870,19 @@ def get_meta(self) -> MetaResponseModel:
 
     ###########################################################################
 
-    def get_table_list(self) -> t.List[str]:
+    def get_table_list(self) -> JSONResponse:
```

Comment:
Will this give us that error we had before in FastAPI?

Reply:
We are fine with this because that bug is fixed in [v0.89.1](https://github.com/tiangolo/fastapi/releases/tag/0.89.1).